packages/data-models/src/language.js

```diff
@@ -0,0 +1,70 @@
+// TODO: This creates a circular dependency and probably prevent Jest tests to succeed
+import { Lang } from './constants.js'
```

Comment:
That's because we import `Lang` into `language.js` from the `constants.js`. `constants.js`, in turn, imports `Language` from `language.js`. I think it is probably a bad design and I need to rethink it.